airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -19,10 +19,11 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/full_refresh_configured_catalog.json"
-      empty_streams: ["external_account_bank_accounts"]
+      empty_streams: ["bank_accounts", "checkout_sessions", "checkout_sessions_line_items", "external_account_bank_accounts"]
     # TEST 1 - Reading catalog without invoice_line_items
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/non_invoice_line_items_catalog.json"
+      empty_streams: ["transfers"]
```

Comment:
What led to these increases in empty streams?

Reply:
We have new account for stripe. It was populated recently by our unblockers team. There are streams which they cannot populate right now for various reasons:
```
bank_accounts
external_account_bank_accounts
checkout_sessions
checkout_sessions_line_items
transfers
```
In order to not to block development of this connector we decided to create a new task to populate those streams later. I can do nothing with it for now

Reply:
Ok, thanks for explaining, as long as that's being tracked!